README.md

```diff
@@ -4,198 +4,25 @@
 [![GitHub issues](https://img.shields.io/github/issues/za419/CadenceBot?style=flat-square)](https://github.com/za419/CadenceBot/issues)
 [![GitHub forks](https://img.shields.io/github/forks/za419/CadenceBot?style=flat-square)](https://github.com/za419/CadenceBot/network)
 
-CadenceBot is a [Discord](https://discordapp.com/) bot, which serves as a client for [Cadence Radio](https://github.com/kenellorando/cadence).
+**CadenceBot** is a [Discord](https://discordapp.com/) bot client for a [Cadence Radio](https://github.com/kenellorando/cadence) stack. 
 
-CadenceBot is fully featured, and is functionally equivalent to the [web client](http://cadenceradio.com/). You can add CadenceBot to your Discord server by clicking [this link.](https://discord.com/api/oauth2/authorize?client_id=372999377569972224&permissions=274881252352&scope=bot)
+It allows you to control your station through a Discord server and even listen to the radio in voice channels! CadenceBot is fully featured and functionally equivalent to the [web client](http://cadenceradio.com/). 
 
-## Implementation and details
+![image](https://user-images.githubusercontent.com/17265041/226133047-2b018a22-32cd-4a2d-b298-48198fe43957.png)
```

Comment:
Is it worth keeping this screenshot in-repo like I did with the diagrams?

Probably not, it's a pretty interchangeable image, but have I established a "convention" by doing so with the SVGs that should now get followed?